homeassistant/helpers/entity.py

```diff
@@ -927,17 +927,19 @@ class ToggleEntity(Entity):
     """An abstract class for entities that can be turned on and off."""
 
     entity_description: ToggleEntityDescription
-    _attr_is_on: bool
+    _attr_is_on: bool | None
```

Comment:
For binary sensor we default this to None. Should we do the same here?

Reply:
Good question, that is existing logic, that fails when it's not explicitly defined (preventing it not being implemented I guess?).

Reply:
Maybe good to force developers to implement `is_on` or `_attr_is_on` and explicitly return None if that is wanted?

Reply:
I don't know. I like the `None` default myself, but didn't implement it here, as this is what is always has been.

That said, before if was only True or False, setting that, would be weird.

I kinda dig the idea of the `None` default.

Reply:
I'm ok with either way, but probably good to do the same way here and in binary sensor.

Reply:
Ok,  will adjust the PR

Reply:
Done in 33552ad